src/component/Banner.tsx

```diff
@@ -106,12 +106,12 @@ const contentContainerStyles = (bodySerif: string) => css`
 `;
 
 const headlineStyles = (headlineSerif: string) => css`
-    font-size: ${headlineSizes.small}rem;
+    font-size: ${headlineSizes.small}px;
```

Comment:
You can use [`remHeadlineSizes`](https://github.com/guardian/source/blob/master/src/core/foundations/src/typography/index.ts#L94) if you prefer 🙂

Reply:
Ah I missed that export! I'm going to stick with this for now as I think we only moved to `rem` as there wasn't a `px` export at the time, plus the final result is the same.